lib/services/irc.rb

```diff
@@ -43,6 +43,7 @@ def receive_gollum
 
   def send_messages(messages)
     messages = Array(messages)
+    messages.push(*Service::DEPRECATION_NOTE.split("\n"))
```

Comment:
It was necessary to split the `DEPRECATION_NOTICE` array up into separate lines, so the IRC command was correctly prepended to each line.